python/setup.py

```diff
@@ -1,2 +1,27 @@
-# Your python setup file. An example can be found at:
-# https://github.com/pypa/sampleproject/blob/master/setup.py
+#!/usr/bin/env python
+# coding=utf-8
+
+from setuptools import setup
+from setuptools import find_packages
+
+
+setup(
+    name="graphframes",
+    version="0.3.0",
```

Comment:
can you update this to 0.5.0  - btw, is there a way to get this programmatically so we don't forget to update this for each release?